unfoldingWord/translationCore
# Copyright Check

#### This pull request addresses:
- Adds copy right check (Front end and Back end)
- Removes params reference from the codebase.

#### How to test this pull request:
- Load a project that has no LICENSE.MD file. (Copyright Check  should open in the stepper)
- Load a project that doesn't have the `license` property in the manifest. json file. (Copyright Check  should open in the stepper)
- Load a project that has both the  LICENSE.MD file and has the `license` property in the manifest. json file. (Copyright Check  should not open in the stepper)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2412)
